docs/install.md

```diff
@@ -42,6 +47,21 @@ Compose can also be installed as a Python package:
 
     $ sudo pip install -U docker-compose
 
+### Container install
+
+Compose can also be run inside a container, from a small bash script wrapper.
+To install compose as a container run:
+
+    curl -L https://github.com/docker/compose/releases/download/1.3.3/compose-run > /usr/local/bin/docker-compose
```

Comment:
What if we added compose-run to the image itself, using docker as the distribution mechanism, and a small install script to echo/output it, e.g.;

```
# To install the script wrapper, run
$ docker run --rm docker-compose install > /usr/local/bin/docker-compose
$ chmod +x /usr/local/bin/docker-compose

# Validate it's working
$ docker-compose --version
docker-compose version: 1.4.0rc3
```